src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,33 @@
+package domain
+
+import domain.CoordinateState.EMPTY
+import domain.constant.Constant.BOARD_SIZE
+import domain.rule.ExactlyFive
+import domain.rule.ExceedFive
+import domain.rule.ForbiddenFour
+import domain.rule.ForbiddenThree
+
+class Board(
+    private val _board: List<MutableList<CoordinateState>> =
+        List(BOARD_SIZE) { MutableList(BOARD_SIZE) { EMPTY } },
+) {
+    val board: List<List<CoordinateState>> get() = _board
```

Comment:
```suggestion
class Board(
    board: List<List<CoordinateState>> =
        List(BOARD_SIZE) { MutableList(BOARD_SIZE) { EMPTY } },
) {
    private val _board: List<MutableList<CoordinateState>> = board.map { it.toMutableList() }
```
생상자는 가급적이면 언더바(`_`) 사용을 지양합니다.
언더바는 주로 backing property를 사용할 때 사용합니다.

Reply:
넵 생성자로 받지 않고 backing property로 두었습니다!